src/hera/__init__.py

```diff
@@ -31,15 +31,15 @@
     get_global_api_version,
```

Comment:
Any chance `expr` can be exposed here as well? `from hera.expr import C, P, ...`

Reply:
We can expose them here as well. Since they were fairly short named variables and come from a well scoped part of hera, having it scoped to `expr` makes it easy to namespace them/know where they came from.

For eg.

```py
import hera.expr as expr

expr.it
expr.g
expr.C
```

vs

```py
import hera

hera.it
hera.g
hera.C
```

LMK what you think.

Reply:
Perhaps this is another instance in which something is a bit unclear and we should leave as is in v0 and welcome feedback from the community. If someone would love to have this exposed at the hera package init level, a PR will be welcome :)